api/src/prescription/learner-management/domain/usecases/reconcile-common-organization-learner.js

```diff
@@ -0,0 +1,47 @@
+import { ReconcileCommonOrganizationLearnerError } from '../errors.js';
+
+/**
+ * @name reconcileCommonOrganizationLearner
+ * @param {Object} params
+ * @param {string} params.campaignCode
+ * @param {number} params.userId
+ * @param {Object} params.reconciliationInfos
+ *
+ * @returns {Promise<void>}
+ */
+const reconcileCommonOrganizationLearner = async function ({
+  campaignCode,
+  userId,
+  reconciliationInfos,
+  campaignRepository,
+  organizationFeatureApi,
+  organizationLearnerImportFormatRepository,
+  organizationLearnerRepository,
+}) {
+  const campaign = await campaignRepository.getByCode(campaignCode);
+  if (!campaign) {
+    throw new ReconcileCommonOrganizationLearnerError('CAMPAIGN_NOT_FOUND');
+  }
+
+  const features = await organizationFeatureApi.getAllFeaturesFromOrganization(campaign.organizationId);
+  if (!features.hasLeanersImportFeature) {
+    throw new ReconcileCommonOrganizationLearnerError('MISSING_IMPORT_FEATURE');
+  }
+
+  const importFormat = await organizationLearnerImportFormatRepository.get(campaign.organizationId);
+  if (!importFormat) {
+    throw new ReconcileCommonOrganizationLearnerError('IMPORT_FORMAT_NOT_FOUND');
+  }
+
+  const userReconciliated = await organizationLearnerRepository.reconcileCommonOrganizationLearner({
```

Comment:
**To do :** Nous souhaiterions déplacer la logique métier dans un model. Le usecase est un chef d'orchestre.
Le `reconcileCommonOrganizationLearner` a ici trop de responsabilité. Il peut être découpé en 2 méthodes : `findByReconciliationInfo` et `save`